docs/players/creating-bots/agent-builder.mdx

```diff
@@ -145,7 +203,34 @@ Clicking the "Validate Code" button will cause a text box to appear with GPT's a
 
 ![Validate Code](./img/agent-builder/validate-code.png)
 
-You can close these results at any time, and reopen them by clicking the clipboard icon.
+Let's give this a try with some code that we know is inefficient.
+The following code attempts to find all the rabbits and chickens nearby, but it searches an unnecessarily large area.
+Copy and paste it into your node:
+
+```typescript
+class LeafNode extends TreeNode {
+	public override async execute(): Promise<NodeStatus> {
+		const bot = this.getData<RGBot>('bot');
+
+		const attackableAnimals = await bot.findEntities({
+            entityNames: ['Rabbit', 'Chicken'],
+            attackable: true,
+            maxDistance: Infinity
+		});
+
+		if (attackableAnimals.length == 0) {
+			return NodeStatus.FAILURE;
+		}
+
+		return NodeStatus.SUCCESS;
+	}
+}
+```
+
+When you click the "Validate Code" button,
+GPT should warn you that `maxDistance` should be set to `100` or less, rather than `Infinity`.
+
+You can close validation results at any time, and reopen them by clicking the clipboard icon.
```

Comment:
```suggestion
You can close validation results at any time and reopen them by clicking the clipboard icon.
```
The comma could be left in if you feel having the break in the sentence is better (even if grammatically incorrect)